pwndbg/commands/version.py

```diff
@@ -189,7 +195,20 @@ def bugreport(run_browser=False):
     github_issue_url = "https://github.com/pwndbg/pwndbg/issues/new"
     github_issue_body = "?body=" + quote(issue_bugreport)
 
-    if run_browser:
+    if use_gh:
+        try:
+            with NamedTemporaryFile("w", delete=True) as f:
+                f.write(issue_bugreport)
+                f.flush()
+                if "EDITOR" in os.environ:
+                    editor = os.environ["EDITOR"]
+                else:
+                    editor = "vi"
+                check_call([editor, f.name])
+                check_call(["gh", "issue", "create", "--body-file", f.name])
+        except Exception:
+            print(please_please_submit + github_issue_url)
```

Comment:
I'm wondering if we should we re-throw the exception after printing this, so that the user understands something went wrong with the GH CLI and it's easier for us to debug.